api/handlers/containerhandler.py

```diff
@@ -346,6 +349,8 @@ def get_all(self, cont_name, par_cont_name=None, par_id=None):
                 result = containerutil.get_stats(result, cont_name)
             result = self.handle_origin(result)
             modified_results.append(result)
+            if self.is_true('phi'):
+                self.log_user_access(AccessType.view_container, cont_name, result.get('_id'))
```

Comment:
We'll probably want to create a more efficient way to call this with a list. Right now it will recreate the hierarchy from this container up. If this level is acquisitions, it means grabbing the same session-project-group from the db for each acquisition, even when it's the same session-project-group.